tests/HelpersTest.php

```diff
@@ -25,14 +25,14 @@ public function testKsortr()
     {
         $unsorted = [
             'zed' => [
-                '1' => '2',
                 'red' => '4',
                 'fed' => 0,
+                '1' => '2',
             ],
             'yo' => [
-                '3' => 'hello',
                 'goo' => 'foo',
                 'bar' => 'baz',
+                '3' => 'hello',
             ],
         ];
```

Comment:
this doesn't seem right?

Reply:
Oh yeah, the ksort test broke on my distro's php version but not my docker container's and I was messing with it but didn't intend to commit this change.